docs/conf.py

```diff
@@ -57,7 +57,7 @@
 
 # General information about the project.
 project = u'Brewtils'
-copyright = u"2018, Logan Asher Jones"
+copyright = u"2018 - 2021, Logan Asher Jones"
```

Comment:
I'm not sure who to attribute this to at this point.  I figured I should at least update the date.

Reply:
Hmm, that's probably a Real Question. The license for everything is MIT, not sure if that matters.

Reply:
I've posed the question to the appropriate folks, but probably won't hold up this PR pending the answer. We can circle back and update this once we get some clarity.